projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -70,7 +70,7 @@ export class AuthService implements OnDestroy {
         concatMap((isCallback) =>
           iif(
             () => isCallback,
-            defer(() => this.handleRedirectCallback()),
+            this.handleRedirectCallback(),
```

Comment:
Removed this `defer` as it's redundant - `handleRedirectCallback` already does it internally and just returns an observable.